build-system/tasks/visual-diff/index.js

```diff
@@ -161,6 +190,7 @@ async function launchPercyAgent(browserFetcher) {
     return;
   }
 
+  // @ts-ignore Type mismatch in library
```

Comment:
It seems like *all* of the Percy library is type-mismatched. Maybe instead of adding its `@types`, can we define all of the types where we import them in a JSDoc comment and add a TODO to fix this once the upstream library is fixed?

Reply:
Figured out how to fix this. The types are built in to the main library at `@percy/core/types/index.d.ts` under the `default` export. Changing the `require` statement as follows did the trick.

```js
const {default: Percy} = require('@percy/core');
```

Unfortunately, the same thing cannot be done for `percySnapshot` because there indeed is a mismatch built in to the library. There's no way to override these errors via JSDoc / global declarations, so we'll have to suppress them until they're fixed upstream.

![image](https://user-images.githubusercontent.com/26553114/120520060-5f3e4700-c3a1-11eb-86c1-7a230189e639.png)